packages/core/src/RenderingEngine/VideoViewport.ts

```diff
@@ -214,24 +214,42 @@ class VideoViewport extends Viewport implements IVideoViewport {
     };
   }
 
+  /**
+   * This is a wrapper for setVideo to allow generic behaviour
+   */
+  public setImageIds(
+    _groupId: string,
```

Comment:
It seems a bit odd to have `_groupId` here, as the method name is `setImageIds`. Perhaps it should be renamed for clarity?

Reply:
I wasn't sure what to call it - that is where the volume id goes if you have a group of things.  I could call it displaySetId?  We need a consistent naming of the thing, but I'm really just not quite sure what to call it.

Reply:
setDataIds as agreed, with a second options argument that has everything else.